From: Ghislain Antony Vaillant <ghisvail@gmail.com>
Date: Mon, 1 Feb 2016 23:08:52 +0000
Subject: Detect fftw libraries using pkgconfig.

---
 setup.py | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

--- a/setup.py
+++ b/setup.py
@@ -121,8 +121,10 @@
         libraries = ['libfftw3-3', 'libfftw3f-3', 'libfftw3l-3']
 
     else:
-        libraries = ['fftw3', 'fftw3f', 'fftw3l', 'fftw3_threads',
-                     'fftw3f_threads', 'fftw3l_threads']
+        import pkgconfig
+        libraries = ['fftw3', 'fftw3_threads', 'fftw3f', 'fftw3f_threads']
+        if pkgconfig.exists('fftw3l'):
+            libraries.extend(['fftw3l', 'fftw3l_threads'])
 
     return libraries
 
