Description: Fix version tag

--- pyfftw-0.11.1.1.orig/pyfftw/_version.py
+++ pyfftw-0.11.1.1/pyfftw/_version.py
@@ -23,7 +23,7 @@ def get_keywords():
     # setup.py/versioneer.py will grep for the variable names, so they must
     # each be defined on a line of their own. _version.py will just call
     # get_keywords().
-    git_refnames = " (tag: v0.11.1)"
+    git_refnames = " (HEAD -> master, tag: v0.11.1)"
     git_full = "09ecbe3f864eb829241eebf990d4332c4164bb1b"
     git_date = "2018-11-19 10:41:27 -0500"
     keywords = {"refnames": git_refnames, "full": git_full, "date": git_date}
